First commits (#5)
cisagov/cw-alarm-sns-tf-module: README.md

| Name | Type |
|------|------|
| [aws_sns_topic.cloudwatch_alarm](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sns_topic) | resource |
| [aws_sns_topic_subscription.account_email](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sns_topic_subscription) | resource |
| [aws_caller_identity.this](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/caller_identity) | data source |
| [aws_organizations_organization.org](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/organizations_organization) | data source |

## Inputs ##

| Name | Description | Type | Default | Required |
|------|-------------|------|---------|:--------:|
| topic\_display\_name | The display name of the SNS topic. | `string` | `"cloudwatch_alarms"` | no |
| topic\_name | The name of the SNS topic. | `string` | `"cloudwatch-alarms"` | no |

## Outputs ##

| Name | Description |
|------|-------------|
| sns\_topic | The SNS topic to which a message is sent when a CloudWatch alarm is triggered. |

## Notes ##

Running `pre-commit` requires running `terraform init` in every directory that

> contains Terraform code. In this repository, these are the main directory and
> every directory under `examples/`.

Reviewer: ```suggestion
contains Terraform code. In this repository,  this is only the main directory.
```
